frontend/src/components/layout/nav.module.css

```diff
@@ -5,30 +5,30 @@
     justify-content: space-between;
 }
 
-.list{
+.listContainer{
     padding-top: 4%;
```

Comment:
As for me it would be easier to understand `rem` or `em` instead of `%`

Reply:
will go ahead and make these changes as well!